python/scipp.cpp

```diff
@@ -6,18 +6,24 @@
 
 namespace py = pybind11;
 
+void init_comparison(py::module &);
+void init_counts(py::module &);
 void init_dataset(py::module &);
 void init_detail(py::module &);
 void init_dtype(py::module &);
 void init_eigen(py::module &);
-void init_counts(py::module &);
+void init_element_array_view(py::module &);
+void init_event_list(py::module &);
 void init_groupby(py::module &);
+void init_geometry(py::module &);
+void init_histogram(py::module &);
 void init_neutron(py::module &);
 void init_operations(py::module &);
-void init_event_list(py::module &);
+void init_reduction(py::module &);
+void init_trigonometry(py::module &);
+void init_unary(py::module &);
 void init_units_neutron(py::module &);
 void init_variable(py::module &);
-void init_element_array_view(py::module &);
```

Comment:
Unfortunately this cannot simply be sorted alphabetically, this may lead to import errors. But maybe those are gone with the restructuring (would know why though?)?

Reply:
i thought the declaration could be in alphabetical order, just the calls below had to be in the right order? Or am I wrong?

Reply:
Ah, sorry, I wasn't reading properly, you are right of course.